README.md

```diff
@@ -356,13 +356,13 @@ curl -X POST -k http://localhost:8081/v1/control/createblockchains -d '{"pluginD
 avalanche-network-runner control create-blockchains '[{"vm_name":"'$VM_NAME'","genesis":"'$GENESIS_PATH'", "subnet_id": "'$SUBNET_ID'"}]' --plugin-dir $PLUGIN_DIR
 ```
 
-To create a blockchain with a subnet id, and both chain config and network upgrade file paths (requires network restart):
+To create a blockchain with a subnet id, and chain config, network upgrade, subnet config file paths (requires network restart):
```

Comment:
Nit: when there are multiple elements in a sentence, "and" is usually put before the last one.